lib/ae_mdw/db/sync/contract.ex

```diff
@@ -52,6 +53,42 @@ defmodule AeMdw.Db.Sync.Contract do
     :ok
   end
 
+  @spec child_contract_mutations(
+          boolean(),
+          Contract.fun_arg_res_or_error(),
+          Txs.txi(),
+          Txs.tx_hash()
+        ) ::
+          {[Mutation.t()], Contract.aex9_meta_info() | nil}
```

Comment:
Perhaps a smaller type signature for this would be `{:ok, [Mutation.t()], Contract.aex9_meta_info()} | :not_found`, since there's no case for `{:ok, [Mutation.t()], Contract.aex9_meta_info()}` where `[Mutation.t()]` isn't empty

Reply:
The problem is that the aex9 might be present or not when the origin mutations is not empty. 
from 
`{:ok, [Mutation.t()], Contract.aex9_meta_info()} | :not_found`
we would need
`{:ok, [Mutation.t()], Contract.aex9_meta_info() | nil} | :not_found`
that would look more complex than:
`{[Mutation.t()], Contract.aex9_meta_info() | nil}`

Reply:
I hadn't realized about that case where there could be mutations and the info would still be `nil`. Thanks for clearing that up!